### DISCUSSION TITLE
elasticsearch - add node_role "transform"

### DISCUSSION THREAD

**Discussion by oestreicher42** [Security-Onion-Solutions/securityonion#10349] [2.4]
I try to add the transform role to my elasticsearch node:

(minion pillar)
elasticsearch:
  node_role:
    - data
    - ingest
    - transform

Unfortunately this does not work.

Is there any other way to do this?
Thanks

**Comment by cm-ops** [comment]
It get set here - https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/elasticsearch/config.map.jinja so you could copy that to the local path and add the transform role to that.

**Reply by oestreicher42** [reply]
Thanks. This worked fine.
Now I could enable the elastic defender integration through kibana.
Perhaps it would be a good idea to enable this by default?
